test/bdd/pkg/didexchange/didexchange_sdk_steps.go

```diff
@@ -297,7 +297,7 @@ func resolveDID(vdriRegistry vdriapi.Registry, did string, maxRetry int) (*diddo
 }
 
 // RegisterSteps registers did exchange steps
-func (d *SDKSteps) RegisterSteps(s *godog.Suite) {
+func (d *SDKSteps) RegisterSteps(s *godog.Suite) { //nolint dupl
```

Comment:
just  for info what warning, linter is throwing here ??

Reply:
Linter thinks that SDK steps and Controller steps are the same. It is false alert. They are not the same. @rolsonquadras  had similar problem earlier.